lib/features/parkings_view/parkings_view.dart

```diff
@@ -27,15 +27,18 @@ class ParkingsView extends ConsumerWidget {
       mapControllers: parkingsMapControllers,
       mapTileBuilder: ParkingTile.new,
       markerBuilder: (item, ref, {required bool isActive}) => Marker(
-        consumeTapEvents: true,
-        markerId: MarkerId(item.id),
-        position: item.location,
-        icon: isActive
-            ? MapMarkerUtils.activeParkingMapMarker
-            : MapMarkerUtils.parkingMapMarker,
-        onTap: () {
-          ref.read(parkingsMapControllerProvider.notifier).onMarkerTap(item);
-        },
+        alignment: Alignment.topCenter,
+        child: GestureDetector(
```

Comment:
Looks nice. Did you think about some splash effect here? Could improve UX

Reply:
I like the idea

Reply:
but not sure how this would look, opening up a task for the future reference